### DISCUSSION TITLE
Getting error =backupstoragelocations.velero.io \ "default\" not found

### DISCUSSION THREAD

**Discussion by tariq1278** [vmware-tanzu/velero#3319] [Community support Q&A] [answered]
Hello Everyone,.I have setup velero on aks azure but getting error =backupstoragelocations.velero.io \ "default\" not found

**Comment by carlisia** [comment]
Hi @tariq1278. What is the output of `velero get backup-location`?

**Comment by carlisia** [comment, marked as answer]
As of Velero versions prior to v1.6, Velero expects a backup storage location (BSL) named "default" will exist and it will use that as the default BSL. In other words, it will use that unless you specify a BSL name during backup creation, OR configure the server to expect a different name as the default.

Running `velero get backup-location` will let you see the names of all your BSLs.

You may edit the backupstoragelocations.velero.io/<some-name>  to be named backupstoragelocations.velero.io/default, this would be a quick fix to this. Alternatively, when you run the backup create, pass the name of the whichever BSL you want to use.

With Velero v1.6 and above, you will be able to set any BSL as the default by using the CLI. Example: `velero backup-location set --default <some-name>`.
